app/controllers/volunteer_applications_controller.rb

```diff
@@ -15,9 +15,8 @@ def create
     @volunteer = Volunteer.new(volunteer_params)
     authorize :volunteer_application, :create?
     if @volunteer.save
-      if VolunteerEmail.active_mail.present?
-        VolunteerMailer.welcome_email(@volunteer, VolunteerEmail.active_mail).deliver
-      end
+      email = EmailTemplate.active.signup.last
+      VolunteerMailer.welcome_email(@volunteer, email).deliver if email
```

Comment:
What happens if `email` doesn't exist? AFAIS `redirect_to thanks_volunteer_applications_url` will happen anyway, so the volunteer get's no email but a confirmation.

Reply:
I think fixing this would be out of scope. In the edge case, the sending would have failed, the Volunteers application would still have succeeded. So it's an edge case and it wouldn't completely brake things. So I'm taking that back...